src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -1011,7 +982,7 @@ def has_args(call_node: ast.Call, required_arg_ids: Tuple[str, ...]) -> bool:
 
 def merge_entry_points(
     *all_entry_points_with_descriptions_of_source: Tuple[str, Dict[str, Dict[str, str]]]
-) -> Dict[str, List[str]]:
```

Comment:
Changed signature, to stay true to our "normalized" structure for entry points. The `List[str]` form for entry points is applied at the last moment when outputting for the setup kwargs.

Reply:
Good change!